ios-swift/App/realmApp.swift

```diff
@@ -12,5 +12,7 @@ var realmApp: App = {
     let data = NSData(contentsOfFile: path)! as Data
     let realmPropertyList = try! PropertyListSerialization.propertyList(from: data, format: nil) as! [String: Any]
     let appId = realmPropertyList["appId"]! as! String
-    return App(id: appId)
+    let baseUrl = realmPropertyList["baseUrl"]! as! String
+    return App(id: appId,
+               configuration: AppConfiguration(baseURL: baseUrl, transport: nil, localAppName: nil, localAppVersion: nil))
```

Comment:
Do we need to specify this baseUrl? Doesn't it already default to `realm.mongodb.com`?

Reply:
Yes but this just allows the devs to use realm-dev or whatever.